system-int-test/test/cmr/system_int_test/site/holdings_provider_test.clj

```diff
@@ -71,6 +71,18 @@
                                    {:granule-ur "Granule1"
                                     :beginning-date-time "1970-06-02T12:00:00Z"
                                     :ending-date-time "1975-02-02T12:00:00Z"}))
+        
+        _g1-online_1 (data/ingest "PROV1" (dg/granule-with-umm-spec-collection
```

Comment:
why the prepending underscore? also, the last one should be a dash

Reply:
- the prepending underscore was already there in the other variables before I started writing
- now edited last underscore to dash